#player, ai 's action
#get 3 coin = 0
#detail need 3 number (coin type)
#get 2 coin = 1
#detail need 1 number (coin type)
#buy card = 2
#detail need 2 number (level, order)

string name
int32 action
int32[] details
int32[] abandon
---
bool success
